src/java_bytecode/Makefile

```diff
@@ -38,8 +38,18 @@ include ../common
 
 CLEANFILES = java_bytecode$(LIBEXT)
 
+clean : clean_library
+
+.PHONY: clean_library
+clean_library :
+	$(MAKE) clean -C library
+
 all: java_bytecode$(LIBEXT)
 
+library/java_core_models.inc:
+	$(MAKE) -C library
```

Comment:
I don't think this will work: the default target in that Makefile is the first one, which is `$(BINARY_DIR)`. I believe you will need to add `java_core_models.inc` to this line here.

Reply:
Isn't the default target
all: java_core_models.inc
?

Reply:
No, it's the first target in a Makefile (except for those starting with a `.`): https://www.gnu.org/software/make/manual/html_node/Goals.html